### PR TITLE
Refactored job commands to use coordinator for SQL jobs

### PR DESCRIPTION
**Pull request by yuce**

### CODE REVIEW

File: base/commands/job/common.go [hunk 1]
```diff
@@ -60,89 +63,91 @@ func idToString(id int64) string {
...
+			coord = conns[0].MemberUUID()
+		}
+		return jid, j.TerminateJob(ctx, jid, terminateMode, coord)
 	})
 	if err != nil {
 		return err
 	}
 	stop()
+	verbose := ec.Props().GetBool(clc.PropertyVerbose)
+	err = nil
```
Comment (kutluhanmetin) [inline base/commands/job/common.go:108]: I think this is a leftover

Comment (kutluhanmetin) [inline base/commands/job/common.go:108]: We already returned the error if it is not nil, we don't need to set it to nil in here.

File: base/commands/job/common.go [hunk 2]
```diff
@@ -60,89 +63,91 @@ func idToString(id int64) string {
...
+			}
+			coord = conns[0].MemberUUID()
+		}
+		return jid, j.TerminateJob(ctx, jid, terminateMode, coord)
 	})
 	if err != nil {
 		return err
 	}
 	stop()
+	verbose := ec.Props().GetBool(clc.PropertyVerbose)
```
Comment (kutluhanmetin) [inline base/commands/job/common.go:107]: I think we don't need to initialize verbose, we can directly use `ec.Props().GetBool(clc.PropertyVerbose)`. I think it is more readable. Same for `wait := ec.Props().GetBool(flagWait)` as well.

File: base/commands/job/common.go (hunk 1 above)
Comment (yuce) [reply base/commands/job/common.go:107]: Addressed at: https://github.com/hazelcast/hazelcast-commandline-client/pull/290/commits/d05e883adb10b872c1f6cd9577db28dfcb3bddce

File: base/commands/job/common.go [hunk 3]
```diff
@@ -60,89 +63,91 @@ func idToString(id int64) string {
...
-			return err
+	}
+	if err != nil {
+		if verbose {
+			ec.PrintlnUnnecessary(fmt.Sprintf("Job %sed: %s", name, idToString(jidv.(int64))))
 		}
+		return nil
 	}
-	return nil
+	return err
```
[the remark this answers is not in the document]
Comment (yuce) [reply base/commands/job/common.go]: Addressed at: https://github.com/hazelcast/hazelcast-commandline-client/pull/290/commits/d05e883adb10b872c1f6cd9577db28dfcb3bddce

File: base/commands/job/common.go (hunk 2 above)
Comment (yuce) [reply base/commands/job/common.go]: Addressed at https://github.com/hazelcast/hazelcast-commandline-client/pull/290/commits/d05e883adb10b872c1f6cd9577db28dfcb3bddce and https://github.com/hazelcast/hazelcast-commandline-client/pull/290/commits/1d87b6c2ace40bb051dadb098ffc91ce68fcf73e

**Review by kutluhanmetin — APPROVED** [review]
LGTM!

**Pull request merged**